controllers/users/demoVerifyUser.js

```diff
@@ -25,6 +24,15 @@ module.exports = async (req, res) => {
           message: 'User has banned by admin'
         });
       }
+      if (userData.verified_status !== 'VERIFIED') {
+        return res.status(401).json({
+          code: 401,
+          data: false,
+          is_banned: true,
+          message: 'User is not verified'
+        });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking the `verified_status` of a user and returning an error response if the user is not verified seems correct. However, the `is_banned` field in the response object is set to `true`. This could be misleading as the user is not banned but unverified. Consider changing this field to something more appropriate like `is_verified`.

```diff
-          is_banned: true,
+          is_verified: false,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
fixed